docs/index.md

```diff
@@ -747,8 +747,8 @@ Create machines on [Openstack](http://www.openstack.org/software/)
 
 Mandatory:
 
- - `--openstack-flavor-id`: The flavor ID to use when creating the machine
- - `--openstack-image-id`: The image ID to use when creating the machine.
+ - `--openstack-flavor-id` or `openstack-flavor-name`: The flavor ID (Name) to use when creating the machine
+ - `--openstack-image-id` or `openstack-image-name`: The image ID(Name) to use when creating the machine.
```

Comment:
For consistency with the rest of the documentation, i suggest :
 * "Identify the flavor that will be used for the machine"
 * "Identify the image that will be used for the machine"